.circleci/config.yml

```diff
@@ -2,39 +2,44 @@ version: 2.1
 orbs:
   codecov: codecov/codecov@3.2.2
   node: circleci/node@5.0.2
+  browser-tools: circleci/browser-tools@1.3.0
 
 jobs:
   test:
     parameters:
       test-type:
         type: string
+      python-version:
+        type: string
     environment:
       NODE_OPTIONS: --max-old-space-size=4096
     docker:
-      - image: circleci/python:3.9.2-browsers
+      - image: cimg/python:<< parameters.python-version >>-browsers
```

Comment:
For some reason, I was having trouble install ffmpeg in the `circleci/python` images.

The `cimg/python` images are the next generation of the `circleci/python` images and switching fixed the ffmpeg issue.

CircleCI [docs](https://circleci.com/developer/images/image/cimg/python?utm_source=google&utm_medium=sem&utm_campaign=sem-google-dg--uscan-en-dsa-maxConv-auth-nb&utm_term=g_-_c__dsa_&utm_content=&gclid=Cj0KCQjwidSWBhDdARIsAIoTVb3fSSKCqxNZow2KGagY8Cr-5KfmBfB6zDHJWCmDme7j4i69IOHp__waAiy5EALw_wcB)